src/_pytest/terminal.py

```diff
@@ -548,10 +549,11 @@ def pytest_runtest_logstart(
     def pytest_runtest_logreport(self, report: TestReport) -> None:
         self._tests_ran = True
         rep = report
-        res: Tuple[
-            str, str, Union[str, Tuple[str, Mapping[str, bool]]]
-        ] = self.config.hook.pytest_report_teststatus(report=rep, config=self.config)
-        category, letter, word = res
+        Res = namedtuple("Res", ["category", "letter", "word"])
```

Comment:
If we add this, let's use a typing.NamedTuple and add expected types + use it in type annotation and internally as well

Reply:
Didn't know `typing.NamedTuple` was a thing, thanks 👍 I've had a go at updating it now